src/utils.jl

```diff
@@ -8,14 +8,15 @@ Implemented values:
 - REN: renewable assets
 - BATT: battery components
 - CONV: battery converters
+- MARK: market type
 """
-@enum ASSET_TYPE LOAD=0 REN=1 BATT=2 CONV=3
+@enum ASSET_TYPE LOAD=0 REN=1 BATT=2 CONV=3 MARK=4

```

Comment:
May be worth finding a way to avoid a new component definition, I add some comments here, but let's tackle this as a second step

Reply:
Completed